src/tf/TFClient.js

```diff
@@ -45,15 +64,16 @@ function TFClient(options) {
  *
  * @param tf - the TF message from the server
  */
-TFClient.prototype.processFeedback = function(tf) {
+TFClient.prototype.processTFArray = function(tf) {
   var that = this;
   tf.transforms.forEach(function(transform) {
     var frameID = transform.child_frame_id;
-    if (frameID[0] === '/') {
+    if (frameID[0] === '/')
+    {
       frameID = frameID.substring(1);
```

Comment:
Yes. At least in the current version of Chrome, if you do
```
var trimmed = '/base_link'.trimLeft('/');
console.log(trimmed);
```
yields "/base_link".
The documentation [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/TrimLeft) doesn't mention ```trimLeft()``` taking any parameters, and advises against using it anyway.

Oddly enough, trimming seemed to work in some places but not others - I thought it safest to just use substring everywhere.